core/src/main/resources/hudson/model/AbstractProject/configure-common.jelly

```diff
@@ -48,19 +48,17 @@ THE SOFTWARE.
 
   <p:config-assignedLabel />
 
-  <f:section title="${%Advanced Project Options}">
-    <f:advanced>
-      <p:config-quietPeriod />
-      <p:config-retryCount />
-      <p:config-blockWhenUpstreamBuilding />
-      <p:config-blockWhenDownstreamBuilding />
-      <st:include page="configure-advanced.jelly" optional="true" />
-      <f:entry title="${%Display Name}" field="displayNameOrNull">
-         <f:textbox checkUrl="'${rootURL}/checkDisplayName?displayName='+encodeURIComponent(this.value)+'&amp;jobName='+encodeURIComponent('${h.jsStringEscape(it.name)}')"/>
-      </f:entry>
-      <f:optionalBlock name="keepDependencies" checked="${it.keepDependencies}" title="${%Keep the build logs of dependencies}" help="/help/tasks/fingerprint/keepDependencies.html"/>
-    </f:advanced>
-  </f:section>
+  <f:advanced>
+    <p:config-quietPeriod />
+    <p:config-retryCount />
+    <p:config-blockWhenUpstreamBuilding />
+    <p:config-blockWhenDownstreamBuilding />
+    <st:include page="configure-advanced.jelly" optional="true" />
+    <f:entry title="${%Display Name}" field="displayNameOrNull">
+       <f:textbox checkUrl="'${rootURL}/checkDisplayName?displayName='+encodeURIComponent(this.value)+'&amp;jobName='+encodeURIComponent('${h.jsStringEscape(it.name)}')"/>
+    </f:entry>
+    <f:optionalBlock name="keepDependencies" checked="${it.keepDependencies}" title="${%Keep the build logs of dependencies}" help="/help/tasks/fingerprint/keepDependencies.html"/>
+  </f:advanced>
```

Comment:
@gusreiber I was not asking whether the change is desirable. I was asking why it is in _this PR_, which is supposed to be about the _New Item_ page. The above diff hunk seems to be adjusting behavior of a configuration screen, which does not seem to be related at all, unless I am missing some subtle interaction.